Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -475,7 +475,7 @@ private func == (lhs: (some StringProtocol)?, rhs: some StringProtocol) -> Bool
      return lhs == rhs
  }
 
-private extension Sequence {
+package extension Sequence {
```

Comment:
@d-ronnqvist Are we able to use `package` visibility now? I don't remember the progress of the Swift language version upgrade.

Reply:
Yes looks like `internal` is enough. (FYI We did step forward to Swift 5.9 compatibility in #861 and adopted the use of `package` in various places in f1ca44cc4cfc88a4f1b57ea9c65ec0c95cdecd7e.

Reply:
PathHierarchy+Find.swift  uses `singleMatch` twice, private to that file, and now this PR calls it as well from PathHierarchy+Error.swift.

Reply:
I've refactored and moved things around and now there's no need to adjust the visibility of this extension.